tests/unit/ui/components/TestRating.js

```diff
@@ -27,6 +40,19 @@ function makeFakeEvent() {
   };
 }
 
+function signInAndDispatchSavedReview({ siteUserId, reviewUserId }) {
+  dispatchSignInActions({ store, userId: siteUserId });
+  const review = {
+    ...fakeReview,
+    user: {
+      ...fakeReview.user,
+      id: reviewUserId,
+    },
+  };
+  store.dispatch(setReview(review));
+  return store.getState().reviews.byId[review.id];
```

Comment:
This works but it's unnecessary since the component takes a review object directly. It would be better like this:

```js
import { denormalizeReview } from 'amo/actions/reviews';

const review = {
  ...fakeReview,
  user: { ... },
};
return denormalizeReview(review);
```

A more common pattern you'll see for this type of test setup would look like `createInternalAddon(fakeAddon)`. The pattern is to use a common helper to simulate what the reducer would be doing without executing reducer code.

You'll have to forgive the name, I'd like to [change it](https://github.com/mozilla/addons-frontend/issues/3219) eventually.